<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>600</width>
    <height>400</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MTP Device Manager</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QLabel" name="deviceInfoLabel">
      <property name="text">
       <string>No device connected / Press Refresh</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QListWidget" name="fileListWidget"/>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QPushButton" name="refreshButton">
        <property name="text">
         <string>Refresh Device</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="createDirButton">
        <property name="text">
         <string>Create Directory</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="deleteDirButton">
        <property name="text">
         <string>Delete Directory</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="writeFileButton">
        <property name="text">
         <string>Write File</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="readFileButton">
        <property name="text">
         <string>Read File</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="deleteFileButton">
        <property name="text">
         <string>Delete File</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>600</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
